src/main/java/com/ldtteam/common/network/AbstractUnsidedPlayMessage.java

```diff
@@ -0,0 +1,36 @@
+package com.ldtteam.common.network;
+
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.world.entity.player.Player;
+import net.neoforged.fml.LogicalSide;
+
+/**
+ * Bidirectional message
+ */
+public abstract class AbstractUnsidedPlayMessage extends AbstractPlayMessage<Player> implements
```

Comment:
Perhaps `BothSided` rather than `Unsided`?

Reply:
If other agrees sure (= im not sure..)